docs/source/routing/self-hosted/containerization/azure.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Deploying GraphOS Router to Azure
+subtitle: Deploy router with Azure Container App
+description: Build and deploy Apollo GraphOS Router as an Azure Container App
+---
+
+Learn how to deploy the router for development on Azure as a Container App.
```

Comment:
Presumably terms like "Container App" are familiar to the users reading these pages. Still wondering if it might be helpful to add them to the glossary for easier outlinking?